source/language/classical.rst

```diff
@@ -427,6 +427,278 @@ Terminating the program early
 The statement ``end;`` immediately terminates the program, no matter what scope
 it is called from.
 
+
+The Switch statement
+--------------------
+
+An OpenQASM3 ``switch`` statement shall use the following keywords:
+    - ``switch``
```

Comment:
From this point on the list is rendering incorrectly. It may be due to an indentation error, [see](https://github.com/openqasm/openqasm/blob/2ed64cbf4a5cb8ebe29fae11d8a4f0a82f8ca790/source/language/classical.rst).

Reply:
Done.